src/Components/Movie/MovieCardMenu.tsx

```diff
@@ -4,25 +4,30 @@ import { MovieActions } from './models'
 const MovieCardMenu: React.FC<MovieActions> = (props) => {
     const [isOpen, setIsOpen] = useState(false)
 
-    const handleClose = () => {
+    const handleOpen = (e: React.MouseEvent) => {
+        e.stopPropagation()
+        setIsOpen(true)
+    }
+
+    const handleClose = (e: React.MouseEvent) => {
+        e.stopPropagation()
         setIsOpen((prevState) => !prevState)
     }
 
-    const handleEdit = () => {
+    const handleEdit = (e: React.MouseEvent) => {
+        e.stopPropagation()
         props.handleEdit()
-        handleClose()
+        setIsOpen((prevState) => !prevState)
     }
 
-    const handleDelete = () => {
+    const handleDelete = (e: React.MouseEvent) => {
+        e.stopPropagation()
```

Comment:
can you call `handleClose` here to reduce repeating code?

Reply:
Fixed